src/utils/browserHistory.js

```diff
@@ -0,0 +1,6 @@
+import {useRouterHistory} from 'react-router';
+import {createHistory} from 'history';
```

Comment:
If the `history` package works without being in the `package.json` then it means it is used by some other package. I guess it's `react-router`, so can't we require `createBrowserHistory` method from `react-router` package, similar as it was done before?